local-setup/scripts/ci-dataloader-v2-regression.py

```
@@ -0,0 +1,696 @@
+        try:
+            uploader._create_boundary_entity(test_tenant, "TEST_BND")
+            # If it returns without error (e.g., service returns different error),
+            # check that it at least didn't silently succeed
+        except PermissionError as e:
+            msg = str(e)
+            assert "role" in msg.lower() or "permission" in msg.lower(), (
+                f"PermissionError message should mention roles. Got: {msg}"
+            )
+            print(f"   PermissionError correctly raised: {msg[:100]}")
+            return
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)